### PR TITLE
fixate 1.14.7 for release

### PR DESCRIPTION
**Pull request by patricklodder**
Fixates the version, tags it as release, sets the release date and copyright year that is displayed in-app. Should be the last commit for the release.

drafted early for visibility.

### CODE REVIEW

File: configure.ac [hunk 1]
```diff
@@ -4,11 +4,11 @@ define(_CLIENT_VERSION_MAJOR, 1)
...
 define(_CLIENT_VERSION_BUILD, 0)
-define(_CLIENT_VERSION_IS_RELEASE, false)
-define(_COPYRIGHT_YEAR, 2022)
+define(_CLIENT_VERSION_IS_RELEASE, true)
+define(_COPYRIGHT_YEAR, 2024)
 define(_COPYRIGHT_HOLDERS,[The %s developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[[Bitcoin Core and Dogecoin Core]])
-define(_CURRENT_RELEASE_DATE,[[1970-01-01]])
+define(_CURRENT_RELEASE_DATE,[[2024-02-27]])
 define(_EXPECTED_DAYS_TO_NEXT_RELEASE,[270])
```
Comment (chromatic) [inline configure.ac:12]: I like this for us.

Comment (patricklodder) [reply configure.ac:12]: We're so screwed 🤣

Comment (slightlyskepticalpotat) [reply configure.ac:12]: just curious, does the `_EXPECTED_DAYS_TO_NEXT_RELEASE` show up anywhere else?

Comment (chromatic) [reply configure.ac:12]: A post-American-Thanksgiving release!

Comment (chromatic) [reply configure.ac:12]: > just curious, does the `_EXPECTED_DAYS_TO_NEXT_RELEASE` show up anywhere else?

See #3191 and #3161 for where this came from and where we intend to use it.

**Comment by patricklodder** [comment]
rebased

**Comment by patricklodder** [comment]
Marking as urgent, to not hold up the tag.

**Comment by chromatic** [comment]
I'm testing this now.

**Review by chromatic — APPROVED** [review]
ACK, all tests pass on x86-64. Let's hold this a couple of hours for other feedback, but I'll target a merge shortly.

**Comment by patricklodder** [comment]
I agree with that approach.

Barring objections, as soon as this is in - and CI passes - a signed tag can be pushed, and we can start the gitian builds; the easy part of the release.

File: configure.ac [hunk 2]
```diff
@@ -4,11 +4,11 @@ define(_CLIENT_VERSION_MAJOR, 1)
...
 define(_CLIENT_VERSION_REVISION, 7)
 define(_CLIENT_VERSION_BUILD, 0)
-define(_CLIENT_VERSION_IS_RELEASE, false)
-define(_COPYRIGHT_YEAR, 2022)
+define(_CLIENT_VERSION_IS_RELEASE, true)
+define(_COPYRIGHT_YEAR, 2024)
 define(_COPYRIGHT_HOLDERS,[The %s developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[[Bitcoin Core and Dogecoin Core]])
-define(_CURRENT_RELEASE_DATE,[[1970-01-01]])
+define(_CURRENT_RELEASE_DATE,[[2024-02-27]])
```
Comment (Saberian1992) [inline configure.ac:11]: [](https://github.com/dogecoin/dogecoin/releases/tag/v1.14.7)

**Comment by georgeartem** [comment]
Release date for 1.14.7 isn’t known ATM.Sent from my iPhoneOn Mar 18, 2024, at 7:28 PM, Saberian1992 ***@***.***> wrote:﻿
@Saberian1992 commented on this pull request.



In configure.ac:
>  define(_COPYRIGHT_HOLDERS,[The %s developers])
 define(_COPYRIGHT_HOLDERS_SUBSTITUTION,[[Bitcoin Core and Dogecoin Core]])
-define(_CURRENT_RELEASE_DATE,[[1970-01-01]])
+define(_CURRENT_RELEASE_DATE,[[2024-02-27]])

File: configure.ac (hunk 1 above)
Comment (MOLAEIA) [inline configure.ac:12]: - [ ] [](https://t.me/NOTAI_App_Bot/notai?startapp=mZhZYFfRirq2PNM)

File: configure.ac [hunk 3]
```diff
@@ -4,11 +4,11 @@ define(_CLIENT_VERSION_MAJOR, 1)
 define(_CLIENT_VERSION_MINOR, 14)
 define(_CLIENT_VERSION_REVISION, 7)
 define(_CLIENT_VERSION_BUILD, 0)
-define(_CLIENT_VERSION_IS_RELEASE, false)
-define(_COPYRIGHT_YEAR, 2022)
```
Comment (MOLAEIA) [inline configure.ac:8]: https://t.me/NOTAI_App_Bot/notai?startapp=mZhZYFfRirq2PNM

**Comment by MOLAEIA** [comment]
[]()

**Comment by MOLAEIA** [comment]
- _****_

File: src/clientversion.h [hunk 4]
```diff
@@ -21,13 +21,13 @@
...
+#define CLIENT_VERSION_IS_RELEASE true
 
 /**
  * Copyright year (2009-this)
  * Todo: update this when changing our copyright comments in the source
  */
-#define COPYRIGHT_YEAR 2022
+#define COPYRIGHT_YEAR 2024
 
 #endif //HAVE_CONFIG_H
```
Comment (MOLAEIA) [inline src/clientversion.h:32]: Checked

**Review by MOLAEIA — COMMENTED** [review]
Checked

File: configure.ac (hunk 2 above)
Comment (MOLAEIA) [reply configure.ac:11]: Checked

Comment (Skandiakiwi13) [reply configure.ac:11]: Checked

File: configure.ac (hunk 1 above)
Comment (Skandiakiwi13) [inline configure.ac:12]: PLK

**Review by Skandiakiwi13 — COMMENTED** [review]
Looks good to me

File: configure.ac (hunk 1 above)
Comment (Skandiakiwi13) [reply configure.ac:12]: E A P E A F

Comment (Skandiakiwi13) [reply configure.ac:12]: PK PF

**Review by Skandiakiwi13 — COMMENTED** [review]
Looks good to me

**Review by Booss13 — COMMENTED** [review]
H

**Pull request merged**